src/lib/swapper/swappers/utils/helpers/helpers.ts

```diff
@@ -251,12 +197,18 @@ export const normalizeIntegerAmount = (amount: string | number | BigNumber): str
 }
 
 export const getApproveContractData = ({
-  web3,
+  approvalAmountCryptoBaseUnit,
+  erc20ContractAddress,
   spenderAddress,
-  contractAddress,
-}: GetApproveContractDataArgs): string => {
-  const contract = new web3.eth.Contract(erc20Abi, contractAddress)
-  return contract.methods.approve(spenderAddress, MAX_ALLOWANCE).encodeABI()
+  web3,
+}: {
```

Comment:
prefer type for return over inline imo

Reply:
ah dang, i just refactored the rest to be inline as I was doing a scroll fest reading them :|

Reply:
ok fixed :)

Reply:
Ah haha, well just an opinion. Your call!